README.md

```diff
@@ -88,12 +88,12 @@ The only required section is `site_info`
 ```
 This section is used to setup your WordPress site for the first time. You can change most of these settings with the WordPress admin once you have logged in after setup.
 
-You can define a specific version of WordPress that you would like to use with `wordpress_version`. If you do not specify, it will use the latest release of WordPress.
+You can define a specific version of WordPress that you would like to use by adding `"wordpress_version": VERSION_NUMBER` on line above the "site_info" section. If you don't add this line, cloud.gov will use the latest release of WordPress.
```

Comment:
That's a good point, especially given how frequently WordPress is released these days. We know `4.4` is tested (and `4.8.1` works in my sandbox) and it'd be good to always start with something we know works and tell people to change it if they want. I'll change this to make it clearer the version in the example is fixed but out of date.